apps/src/lib/ui/ValidationStep.jsx

```diff
@@ -62,7 +62,11 @@ export default class ValidationStep extends Component {
           <div style={style.icon}>{iconFor(stepStatus)}</div>
           <div style={style.headerText}>{stepName}</div>
         </div>
-        {showChildren && <div style={style.body}>{children}</div>}
+        {showChildren && (
+          <div style={style.body} className="validation-children">
```

Comment:
It seems a bit odd to mix both an inline JS style and a SCSS-based one.  Would it be possible to do this completely with inline JS styling?

Reply:
I'm not sure. I played with it and got this far - I added 'div/p' styling to 'style.body'.
![Screenshot from 2022-02-03 11-40-47](https://user-images.githubusercontent.com/2959170/152417211-1057e48d-abda-4df2-89ca-0704ff6258ac.png)

But when I examine the element styling in the dev tools, the 'font-size: 14' is overridden by the styling in application.css.

There might be a way to get this to work, but I don't know it. Any advice?

Reply:
i don't think this (accessing elements using nested selectors) is possible with our current JS-styling setup -- it's one of the many reasons we need to change our approach to styling on the frontend!

Reply:
Agreed.  That said, I had a feeling Radium, which we sometimes use, might have some support for nested selectors?

Reply:
not that i know of, but if you know how to make nested selectors work with radium, let me know! it's still deprecated and we'll need to stop using it, but it may be useful in the meantime while we're transitioning

Reply:
It looks like the nested selectors only work with three psuedo-selectors as described [here](https://github.com/FormidableLabs/radium/blob/master/docs/guides/README.md#browser-states).  

However, for this particular change, it looks like we could have generated the desired styling in JS using the `Style` component described [here](https://github.com/FormidableLabs/radium/tree/master/docs/api#style-component)?